packages/select/src/components/multi-select/multiSelect.tsx

```diff
@@ -294,33 +338,46 @@ export class MultiSelect<T> extends AbstractPureComponent<MultiSelectProps<T>, M
                     // Normally, Popover would also need to attach its own `onKeyDown` handler via `targetProps`,
                     // but in our case we fully manage that interaction and listen for key events to open/close
                     // the popover, so we elide it from the DOM.
-                    onKeyDown: this.getTagInputKeyDownHandler(handleKeyDown),
-                    onKeyUp: this.getTagInputKeyUpHandler(handleKeyUp),
+                    onKeyDown:
+                        this.props.customTarget != null
+                            ? this.getKeyDownHandler(handleKeyDown)
+                            : this.getTagInputKeyDownHandler(handleKeyDown),
+                    onKeyUp:
+                        this.props.customTarget != null
+                            ? this.getKeyUpHandler(handleKeyUp)
+                            : this.getTagInputKeyUpHandler(handleKeyUp),
```

Comment:
seeing the keydown to open the popover seemingly be missed makes me wonder if these should be something like `onKeyDown: this.getTargetKeyDownHandler(handleKeyDown)`

and handle any splits on `customTarget` within

Reply:
hmmm keeping them separate because there is decently separate logic between them, namely how some keystrokes have built in events like space and downarrow (which move the scrollview down).

TagInput is able to automatically ignore it because the target is an input, so space and downarrow will not do it's default behavior